examples/ZMP/LinearInvertedPendulum.m

```diff
@@ -221,22 +243,16 @@
       dt = diff(breaks);
 
       hg = h/9.81;
-      A = [zeros(2),eye(2);zeros(2,4)]; 
-      B = [zeros(2);eye(2)];
-      C = [eye(2), zeros(2)];
-      D = -hg*eye(2);
-
-      % LQR costs for the output system
-      Q = options.Qy(5:6,5:6); % for consistency with the other solution methods
-      R = zeros(2);
-
-      % Convert to costs in xbar, u
-      Q1 = C'*Q*C;
-      R1 = R + D'*Q*D;
+      [A, B, C, D, Q, R, Q1, R1, N] = LinearInvertedPendulum.setupLinearSystem(hg, options.Qy);
```

Comment:
you're calling this twice if use_lqr_cache = true and you get a cache miss.  but i guess that's ok.